### DISCUSSION TITLE
plot_model() graph can not use zoom slicing

### DISCUSSION THREAD

**Discussion by nutyfreshz** [pycaret/pycaret#2372] [Q&A] [answered]
When I used plot_model() on Pycaret in Colabs, I got an unusable graph like this.

![image](https://user-images.githubusercontent.com/98257732/161576474-8604952c-76ee-41a6-bfa5-a883a175be66.png)

How I can use zoom slicing on my graph like this? They just used normal plot_model() and got zoom slicing feature.

![image](https://user-images.githubusercontent.com/98257732/161576712-a9cd2fc8-c915-4b42-b6a1-2a83eac42f4b.png)

Thank you

**Comment by ngupta23** [comment, marked as answer]
When you have too many data points, the interactivity is disabled (because they cause the notebook to freeze). However, you can manually enable interactivity if you that is what you want..

Please refer to [the 104 notebook](https://nbviewer.org/github/pycaret/pycaret/tree/master/examples/time_series/forecasting/) for details: Section **Setting Renderer**. 

You can also refer to the docstring in `setup` or `plot_model` for more details.

**Reply by nutyfreshz** [reply]
thank you so much.

**Reply by Curovearth** [reply]
The link to notebook is not accessible. Could you please update it.
Thanks

**Reply by ngupta23** [reply]
Thanks for pointing it out. I have updated it.
